Cargo.toml

```diff
@@ -22,7 +22,7 @@ serde_json = "1.0.80"
 
 base64 = "0.13.0"
 base64-url = "1.4.13"
-openssl = "0.10.55"
+openssl = { version = "0.10.55", features = ["vendored"]}
```

Comment:
We removed it from paperboy :). Why is it needed here? Compatibility with arm?

Reply:
Yeah finding a prebuilt openssl that can be statically linked to arm with musl is a nightmare

Reply:
Agreed!

Reply:
Done